src/sv-pipeline/05_annotation/scripts/compute_AFs.py

```diff
@@ -181,11 +308,15 @@ def main():
         males = [line.split('\t')[1] for line in famfile if line.split('\t')[4] == '1']
         females = [line.split('\t')[1] for line in famfile if line.split('\t')[4] == '2']
         sexes = 'MALE FEMALE'.split()
+        if args.par is not None:
```

Comment:
Would it make sense to intialize `parbt` to an empty BedTool, ie `parbt = pbt.BedTool("", from_string=True)`? Then you wouldn't have to check if it's `None` around the script, you could just always intersect with it. Might simplify the code a little.

Reply:
Sure, that can work, although I'd prefer to only add the `PAR` definition to the VCF header if a file is passed as `--par`, so we'll still have to check if `len(parbt) > 0` instead of `par is not None:`. Agreed though this is a simpler solution, so I've made those changes.